generators/app/templates/express-api/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM alpine:3.3
```

Comment:
@calebcartwright 
of knowing what other things they add in there, obviously its not that hard as you can hop over to the node image github page and check it out.
We could do that if you want, I typically dont bother with it